### ISSUE TITLE
text.cpp: error: call of overloaded create

### ISSUE THREAD

**Issue by RedyJack** [twistedfall/opencv-rust#505]
Hi I have problem during building of small project containing opencv bindings. Compilation of opencv text.cpp seem to fail. 
1. Operating system 
```
NAME="Ubuntu"
VERSION="20.04.6 LTS
```
2. Open CV installed using `apt-get install libopencv-dev `
3. Open CV version
```
cat /usr/lib/x86_64-linux-gnu/pkgconfig/opencv4.pc 
# Package Information for pkg-config

prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib/x86_64-linux-gnu
includedir_old=${prefix}/include/opencv4/opencv
includedir_new=${prefix}/include/opencv4

Name: OpenCV
Description: Open Source Computer Vision Library
Version: 4.2.0
Libs: -L${exec_prefix}/lib/x86_64-linux-gnu -lopencv_stitching -lopencv_aruco -lopencv_bgsegm -lopencv_bioinspired -lopencv_ccalib -lopencv_dnn_objdetect -lopencv_dnn_superres -lopencv_dpm -lopencv_highgui -lopencv_face -lopencv_freetype -lopencv_fuzzy -lopencv_hdf -lopencv_hfs -lopencv_img_hash -lopencv_line_descriptor -lopencv_quality -lopencv_reg -lopencv_rgbd -lopencv_saliency -lopencv_shape -lopencv_stereo -lopencv_structured_light -lopencv_phase_unwrapping -lopencv_superres -lopencv_optflow -lopencv_surface_matching -lopencv_tracking -lopencv_datasets -lopencv_text -lopencv_dnn -lopencv_plot -lopencv_ml -lopencv_videostab -lopencv_videoio -lopencv_viz -lopencv_ximgproc -lopencv_video -lopencv_xobjdetect -lopencv_objdetect -lopencv_calib3d -lopencv_imgcodecs -lopencv_features2d -lopencv_flann -lopencv_xphoto -lopencv_photo -lopencv_imgproc -lopencv_core
Libs.private: -ldl -lm -lpthread -lrt
Cflags: -I${includedir_old} -I${includedir_new}
```
4. `rustc 1.72.1 (d5c2e9c34 2023-09-13)`
5. Build output attached as log file [build.log](https://github.com/twistedfall/opencv-rust/files/13040533/build.log)

In the code I basically just do this (if this matters):
```
use opencv::prelude::*;
use opencv::core;
use opencv::objdetect;

pub fn detect_faces() {
    let face_cascade = objdetect::CascadeClassifier::new("models/haarcascade_frontalface_default.xml").unwrap();
}
```
I am not sure if this is bug, setup issue or me doing something wrong. Any guidance?

**Comment by twistedfall** [comment]
That does look like a bug, thanks for reporting! Is there any chance you could try checking if the crate version 0.84.5 works for you?

**Comment by RedyJack** [comment]
Thanks for quick response. This issue does not appear on 0.84.5.

**Comment by twistedfall** [comment]
This should be fixed in 0.85.3
